test/units/modules/storage/netapp/test_na_ontap_cluster.py

```diff
@@ -1,198 +0,0 @@
-# (c) 2018, NetApp, Inc
```

Comment:
why does this file show as deleted?

Reply:
Forgot to revert that change in this request, fixed that.